frontend/src/components/atoms/PltAmount.vue

```diff
@@ -65,8 +66,9 @@ const amounts: ComputedRef<[string, string]> = computed(() => {
 	if (props.formatNumber) {
 		const { formatedNum, suffix } = numberFormatter(
 			BigInt(significantDigits),
-			props.decimals
+			props.fixedDecimals ?? 3
```

Comment:
The `PltAmount` component is used at various places. I wouldn't hardcode any default value (meaning 3) randomly in the code base as it is hard to find such a hardcoded value and confusing when someone uses the component being not aware of this default behavior.